src/MagnumPlugins/WavAudioImporter/WavHeader.h

```diff
@@ -31,30 +31,59 @@
 
 #include "Magnum/Types.h"
 
+#define WAVE_FORMAT_PCM 0x0001
```

Comment:
@Squareys @mosra I used defines as that's what they are in the original C code, and I wanted this to be as "close to original" as possible; we could go with making them entirely internal to the source file if you want, to make them more encapsulated.

Or an enum, or even const int's or something. I don't mind. I'd prefer defines for the legacy reason though.